qrest/resource.py

```diff
@@ -585,6 +586,9 @@ def _get(self, extra_request=None, extra_body=None, extra_file=None):
             # This is a back-catcher for HTTP errors that were not caught before. Code shoul
             # not get here
             raise http
+        except requests.Timeout as timeout:
+            # Catch-all for both connection timeout and read timeout
+            raise timeout
```

Comment:
Do you prefer a qrest specific exception instead of the default requests exception?

Reply:
Agreed, makes sense. I will adjust the code.

Reply:
That would be a ```RestTimeoutError(RestClientResourceError)```, right?

Is ```raise RestTimeoutError("Request to client timed out")``` enough, or would you prefer more info in the error message?

Reply:
Some of the other qrest exceptions mention the URL, e.g.
``` python
if self.code == 400:
    raise RestBadRequestError("Bad request for resource %s" % (self.response.url,))
```
The URL seems like useful info for a `RestTimeoutError` too. Apart from that one, possibly the original `requests.Timout` exception contains some useful info too.

Reply:
Done, both the url and the original error message are included now.